ldp/nn/agent/simple_local_agent.py

```diff
@@ -33,6 +34,19 @@ class AgentLMConfig(_LMConfig):
     max_new_tokens: int = 50
     parallel_tool_calls: bool = False
 
+    llm_call_kwargs: dict = Field(
+        default_factory=dict,
+        description="Additional kwargs to pass to LocalLLMCallOp.forward. "
+        "Note that the validator defaults top_k=None and top_p=1.0, which "
+        "are better defaults than HF's.",
+        validate_default=True,
+    )
+
+    @field_validator("llm_call_kwargs")
+    @classmethod
+    def validate_llm_call_kwargs(cls, v: dict) -> dict:
```

Comment:
```suggestion
    def inject_preferred_llm_call_defaults(cls, v: dict) -> dict:
```

I might name this closer to its purpose. It's not really a validator